Amplify/Categories/DataStore/Model/Temporal/README.md

```diff
@@ -0,0 +1,58 @@
+## Custom Data Types
+
+`DataStore > Model > Temporal`
+
+Model-based programming aims to simplify data management on apps and enable developers to focus on the business and UI implementation while the technical aspects on persisting data are abstracted away from the core logic. Therefore, data types are a critical piece of it.
+
+This module provides types that will complement the Swift provided ones when more control is needed over the persisted values.
```

Comment:
This sentence is kinda vague.... and i'm not sure how to improve it because i'm not exactly sure what we are trying to say...

Reply:
I changed a bit, let me know what you think.